CONTRIBUTING.md

```diff
@@ -35,24 +35,127 @@ requests).
+  is accepted, as the reviewer will "squash and merge" your PR once ready.
+  
+- It is the submitters responsibility to ensure that a PR can be cleanly
+  merged into the master branch.
+
+- All code submitted must have at least 60% coverage for the affected lines.
+  Reviewers may request higher coverage, or additional tests generally, if 
+  the changes made warrant this. Generally, aim for a net increase in code
+  coverage for the project with each PR.
+  
+- Code submitted must not drop the overall coverage of the project below 80%.
+  If the code coverage is already below 80%, then contributions must increase
+  the net coverage to be accepted.
+  
+- Code changes that have implications for the specification must have an 
+  associated thread of discussion here on [list][oidf-ac-list], and consensus 
+  must have been reached that the change is worthwhile. Preferably, a separate
+  pull request should already have been merged into to the 
+  [OpenYOLO-Spec repository][spec-repo] before a code change is submitted.
+  
+- Code changes that affect the API must also be demonstrated as part of the 
+  test and sample applications.
+
+- Code changes that affect the SPI must also be demonstrated as part of the 
+  test and demo providers.
+
+- Complex changes _should_ be broken into multiple smaller pull requests for 
+  review, and these should not be directly submitted to the master branch. 
+  Instead, they must be submitted to a feature branch until the complete 
+  set of changes is made, and then merged into master upon agreement that
+  the work is complete.
+
+- As a general guideline, pull requests should not contain more than a 1000
+  line delta. Reviewers may still agree to review PRs that are larger than
+  this at their discretion, or they can request that the PR be split into
+  smaller changes for more effective review.
+
+## Responsibilities of reviewers
+
+Reviewers (members of the 
+[@openid/openyolo-android-maintainers][maintainers-team] team) are expected
+to adhere to the following guidelines:
+
+- New pull requests should be allocated a reviewer within one business day of
+  initial submission.
+
+- Initial review comments on a PR should be supplied within two business days
+  of a reviewer being assigned.
+
+- Reviewers should not review code that is submitted by members of their own
+  organization, unless there is no reasonable alternative.
+
+- Reviewers must squash all PRs into a single commit on submission (use the
+  "squash and merge" option when accepting the pull request).
+
+- All comments and discussion should occur via issues, in reviews, or on the
+  [mailing list][oidf-ac-list]. Avoid direct communication outside of these, 
+  or at least summarize such direct communication in one of the official
+  avenues, in order to ensure everyone has a chance to understand the decision
+  making process.
+
+# Nonclamenture
+
+The following acronyms and terms will be common in reviews:
+
```

Comment:
Nit: IMO I think we can omit this (unless maybe it becomes a problem :)) and instead stick Git's PR. As minor as this is, I think it sets a good precedent to avoid anything being "Google-specific" when possible.

Reply:
Done.